corehq/apps/repeaters/tests/test_repeater.py

```diff
@@ -302,6 +296,7 @@ def test_excluded_case_types_are_not_forwarded(self):
         CaseFactory(self.domain_name).post_case_blocks([non_white_listed_case])
         self.assertEqual(1, len(self.repeat_records(self.domain_name).all()))
 
+    # @run_with_all_backends
```

Comment:
I believe that to be true. The only instance it wouldn't be is if someone hand edited their casexml blocks in their forms to specifically make those different, and it seems pretty unlikely that that would happen.